workitem/link/link_repository_blackbox_test.go

```diff
@@ -131,3 +131,94 @@ func (s *linkRepoBlackBoxTest) TestDisallowMultipleParents() {
 	_, err = linkRepository.Create(s.ctx, Parent2ID, ChildID, s.testTreeLinkTypeID, s.testIdentity.ID)
 	require.NotNil(s.T(), err)
 }
+
+// TestCountChildWorkitems tests total number of workitem children returned by list is equal to the total number of workitem children created
+// and total number of workitem children in a page are equal to the "limit" specified
+func (s *linkRepoBlackBoxTest) TestCountChildWorkitems() {
+	workitemRepository := workitem.NewWorkItemRepository(s.DB)
+
+	// create a parent workitem
+	Parent, err := workitemRepository.Create(
```

Comment:
Yes. The capital letter means its global. I made the changes here ->  	086c187 
Could you please take a look?